### ISSUE TITLE
Improved Differentiation of Log Message Levels

### ISSUE THREAD

**Issue by agileshaw** [canonical/charmed-openstack-upgrader#370]
We should clearly define what each level of log represents to ensure proper differentiation. An suggestion would be:
- **debug**: low-level information which can be used for debugging purposes
- **info**: general information for regular users using COU
- **warning**: an irregularity has been detected, which COU can handle, but its operations may cause disruptions. `--force` option can be used to bypass the checks and force the upgrade.
- **error**: an issue has been detected that requires user intervention, as it is beyond COU's capability to handle autonomously.

With this standard:
- current pre-plan sanity check failures should be considered as errors because there's nothing COU can do to continue with the upgrade.
- non-empty hypervisors detected should raise warnings, and user can choose to use `--force` to upgrade them
- (after #369 is implemented) charms with `latest/stable` channel detected should also raise warnings, and using `--force` will switch its channel to a release-based one during upgrade

**Comment by aieri** [comment]
low priority, closing for now
